### DISCUSSION TITLE
PLUG Tuya TS011F_plug_1 _TZ3000_cehuw1lw non rileva valore sotto 10 watt con zigbee2mqtt

### DISCUSSION THREAD

**Discussion by manny199** [indomus/forum#1965] [GENERALE - Discussioni generiche]
Ciao,
ho delle prese plug Tuya TS011F_plug_1 _TZ3000_cehuw1lw che con zigbee2mqtt non rilevano il valore al di sotto dei 10 watt.
Sotto tale soglia il valore rilevato è ZERO.
Qualcuno si è imbattuto con lo stesso problema?

Grazie

**Comment by snakuzzo** [comment]
Hai letto sulla pagina ad hoc?
È tutto scritto

https://www.zigbee2mqtt.io/devices/TS011F_plug_1.html

Cito...

_Those devices report power, current and voltage unreliably: Changes in any of those metrics sometimes take a few minutes before being reported and constant loads are reported as 0 for a few minutes just to return to their expected values a few minutes later. There is no known workaround for that behaviour._

Magari prova ad aggiornare il device via OTA... è scritto sempre tutto lì su quella pagina

**Reply by manny199** [reply]
Avevo già letto quanto hai scritto ma pensavo si riferisse a tempo fa ... speravo che avessero risolto in qualche modo.
La cosa pazzesca è che non trovo prese zigbee con chip diverso e non saprei veramente cosa acquistare. Ti viene in mente qualcosa di affidabile?

**Reply by snakuzzo** [reply]
Le blitzwolf vanno uno spettacolo

**Reply by manny199** [reply]
Lo so ma dove le trovo? Non stanno da nessuna parte

**Reply by snakuzzo** [reply]
AliExpress, bangood, etc

**Reply by manny199** [reply]
Purtroppo non se ne trovano se non sul sito Blitzwolf a prezzo pieno

**Reply by Scialla** [reply]
Su Banggood stanno a 20,84 spedite

**Comment by manny199** [comment]
Ho comprato 4 blitzwolf BW-SHP13 sperando di risolvere ma anche queste, come le altre che peraltro costavano molto meno, hanno lo stesso identico problema, cioè non rilevano il valore power al di sotto di 10 watt.

**Reply by snakuzzo** [reply]
Non mi risulta proprio...

![Screenshot_20221024-195125.jpg](https://user-images.githubusercontent.com/7205371/197592778-c9048008-fae6-4e5d-8566-a3db283dda7f.jpg)

**Reply by snakuzzo** [reply]
Ricordati di impostare il `poll interval` per interrogare il device per reperire i dati. Se non lo imposti il device non manda niente di sua spontanea volontà

**Comment by manny199** [comment]
![Screenshot_20221024-195357_Chrome](https://user-images.githubusercontent.com/50017898/197593361-3b09be5b-99e0-49bc-8001-e6cc5349faa1.jpg)
Poll interval era già impostato a -1

**Reply by manny199** [reply]
Forse hai un firmware diverso o hai fatto qualche magia. Sul sito di koenkk mi scrivo che é un problema hardware che affligge tutti questi dispositivi. Mi dai i tuoi settaggi?

**Reply by snakuzzo** [reply]
Ma che problema avete? Perché postate sempre aprendo nuovi thread? :D
Dovete postare sotto l'ultimo messaggio seguendo la discussione...e non crearne di nuovi come hai fatto ora.
Ad ogni modo, come puoi vedere dal mio screenshot, il valore `Power` viene rilevato anche quando è inferiore a 10W

**Reply by snakuzzo** [reply]
Il pollo interval va messo ad un valore maggiore di zero, altrimenti non manda nulla... C'è anche scritto

**Reply by manny199** [reply]
Il tuo poll interval è -1?
Hai qualche altro settaggio da suggerire?

**Reply by snakuzzo** [reply]
Se lo imposti a `-1` lo disabiliti

**Reply by snakuzzo** [reply]
Mettilo tipo a 5

**Reply by snakuzzo** [reply]
Io l'ho impostato a 15...quindi ogni 15 secondi manda le letture

**Reply by manny199** [reply]
Mi dai la stringa esatta del poll interval da inserire in configuration.yaml di zigbee2mqtt?

**Reply by snakuzzo** [reply]
Lo puoi fare da webgui di zigbee2mqtt.
Selezioni il device poi vai su settings (specific)

**Reply by manny199** [reply]
Purtroppo su setting specific ho solo calibration

**Reply by manny199** [reply]
Modificato poll interval ma non cambia nulla, sotto i 10 watt sempre zero. Evidentemente anche se hanno stesso nome l'hardware è differente

**Reply by manny199** [reply]
Poiché nella webgui non è presente la relativa sezione, ho inserito in configuration.yaml di zigbee2mqtt il poll interval nel seguente modo:
```
  '0xa5c1344a96rna25x':
    friendly_name: FRIGO
    measurement_poll_interval: 15
```
E' corretto? Se si continua a rilevare zero sotto i 10 watt e la misurazione inizia al di sopra di questo valore

**Reply by snakuzzo** [reply]
Sì è corretto.

**Reply by manny199** [reply]
Hai avuto la fortuna di trovare un plug con hardware diverso. Non sempre le cose più recenti sono migliorative
